ext/standard/mail.c

```diff
@@ -74,6 +74,18 @@ static bool php_mail_build_headers_check_field_value(zval *val)
 			}
 			return FAILURE;
 		}
+		/*
+		 * This is a way of writing that violates rfc2822,
+		 * but there are cases where this is accepted.
+		 */
+		if (*(value->val+len) == '\n') {
```

Comment:
MacOS `\r` alone should not be probably allowed as well.

Reply:
That's okay because it's already not allowed :)

Reply:
is it tested? I mean `\r`, not `\r\n`.

Reply:
I just tested it. `\r` alone is also not allowed.

Reply:
I mean to add it into phpt test.